src/middle/Middle.ml

```diff
@@ -199,3 +167,9 @@ let operator_return_type_from_string op_name argtypes =
 
 let operator_return_type op =
   operator_return_type_from_string (string_of_operator op)
+
+let rec sexp_of_expr_typed_located {expr; _} =
```

Comment:
Is the point of these to not print all the metadata in the sexpressions?

Reply:
yep